### PR TITLE
Add configurations

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configuration/docker_image_project_name.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ss-elections
```
Comment (as2388) [inline configuration/docker_image_project_name.txt:1]:
```suggestion
ssf-elections
```

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    {"SourceKey": "Age (Value) - ssf_elections_demog", "PipelineKey": "age_raw"},
+    {"SourceKey": "Age (Time) - ssf_elections_demog", "PipelineKey": "age_time"},
+    {"SourceKey": "Recently_Displaced (Value) - ssf_elections_demog", "PipelineKey": "recently_displaced_raw"},
+    {"SourceKey": "Recently_Displaced (Time) - ssf_elections_demog", "PipelineKey": "recently_displaced_time"},
+    {"SourceKey": "Household_Language (Value) - ssf_elections_demog", "PipelineKey": "household_language_raw"},
+    {"SourceKey": "Household_Language (Time) - ssf_elections_demog", "PipelineKey": "household_language_time"},
+    {"SourceKey": "District (Value) - ssf_elections_demog", "PipelineKey": "location_raw"},
+    {"SourceKey": "District (Time) - ssf_elections_demog", "PipelineKey": "location_time"}
+  ],
+  "ProjectStartDate": "2021-04-23T08:00:00+03:00",
```
Comment (as2388) [inline configuration/pipeline_config.json:79]: think this needs updating

File: configuration/pipeline_config.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json"
+  },
+  "TimestampRemappings": [
+  ],
+  "SourceKeyRemappings": [
+    {"SourceKey": "avf_phone_id", "PipelineKey": "uid"},
+
+    {"SourceKey": "Rqa_Elections_S01E01 (Text) - ssf_elections_s01e01_activation", "PipelineKey": "rqa_elections_s01e01_raw", "IsActivationMessage": true},
+    {"SourceKey": "Rqa_Elections_S01E01 (Run ID) - ssf_elections_s01e01_activation", "PipelineKey": "rqa_elections_s01e01_run_id"},
+    {"SourceKey": "Rqa_Elections_S01E01 (Time) - ssf_elections_s01e01_activation", "PipelineKey": "sent_on"},
```
Comment (as2388) [inline configuration/pipeline_config.json:44]: do you want to configure the other episodes here yet? (I see that you added them in the other parts of configuration)

**Review by as2388 — APPROVED** [review]

File: configuration/docker_image_project_name.txt (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/docker_image_project_name.txt:1]: updated thanks.

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json:103]: will update in go-live pr

File: configuration/pipeline_config.json (hunk 3 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json:44]: Yup, updated thanks.

**Review by DanielMwendwa — APPROVED** [review]

File: configuration/pipeline_config.json [hunk 4]
```diff
@@ -0,0 +1,122 @@
+{
+  "PipelineName": "ssf-elections",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
```
Comment (DanielMwendwa) [inline configuration/pipeline_config.json:7]: gs://avf-credentials/SSF_REC-PPE-Textit-Token.txt

**Pull request merged**